.gitignore

```diff
@@ -1,19 +1,108 @@
-# Python droppings
+### Python ###
+# Byte-compiled / optimized / DLL files
+__pycache__/
 *.py[cod]
+*$py.class
```

Comment:
That's a new one to me. Is it for Jython?

Reply:
> That's a new one to me. Is it for Jython?

It is, I just copy & pasted from https://www.gitignore.io/api/python :man_facepalming: